src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -225,6 +226,7 @@ public Func<HttpClient> HttpClientFactory
     internal static void RegisterOtlpExporterOptionsFactory(IServiceCollection services)
     {
         services.RegisterOptionsFactory(CreateOtlpExporterOptions);
+        services.RegisterOptionsFactory(configuration => new ExperimentalOptions(configuration));
```

Comment:
@vishweshbankwar Do me a favor and remove this line:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/4cccb09239aff1250967c543f7483aaf88e538a3/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporterHelperExtensions.cs#L407

It shouldn't be needed now that you have this here.

Reply:
My bad, fixed b91a7337f501c28629e3b55daa524156e0a4d0d5